R/data_meredith.R

```diff
@@ -0,0 +1,353 @@
+#'  @examples
+#' # To convert data frame to tidy data (long) format, run:
+#' library(tidyverse)
+#' library(stringr)
+#' candy_rankings_tidy <- candy_rankings %>%
+#'   gather(characteristics, present, -c(competitor_name, sugar_percent, price_percent, win_percent)) %>%
+#'   mutate(present = as.logical(present)) %>%
+#'   arrange(competitor_name)
+"candy_rankings"
+
+
+
+#' Chess Transfers
+#'
+#' The raw data behind the story
+#' "American Chess Is Great Again"
+#' \url{https://fivethirtyeight.com/features/american-chess-is-great-again/}.
+#'
+#' @format A data frame with 932 rows representing international player transfers and 5 variables:
+#' \describe{
+#'   \item{url}{The corresponding website on the World Chess Federation page which details the transfers of a given year.}
+#'   \item{id}{An numeric identifier for the chess player who transfered.}
+#'   \item{federation}{The current national federation of the chess player}
+#'   \item{former_fed}{The national federation from which the chress player has transfered.}
+#'   \item{transfer}{The date at which the transfer took place.}
+#' }
+#' @source World Chess Federation \url{https://ratings.fide.com/fedchange.phtml}
+"chess_transfers"
+
+
+
+#' Congress Generic Ballot Polls 
+#'
+#' The raw data behind the story
+#' "Are Democrats Winning The Race For Congress?"
+#' \url{https://projects.fivethirtyeight.com/congress-generic-ballot-polls/}.
+#'
+#' @format A data frame with 934 rows representing polls and 21 variables:
+#' \describe{
+#'   \item{subgroup}{}
+#'   \item{modeldate}{}
+#'   \item{startdate}{Start date of the poll.}
+#'   \item{enddate}{End date of the poll.}
+#'   \item{pollster}{The organization that conducted the poll (rather than the organization that paid for or sponsored it)}
+#'   \item{grade}{}
+#'   \item{samplesize}{}
+#'   \item{population}{}
+#'   \item{weight}{}
+#'   \item{influence}{}
+#'   \item{dem}{}
+#'   \item{rep}{}
+#'   \item{adjusted_dem}{}
+#'   \item{adjusted_rep}{}
+#'   \item{multiversions}{}
+#'   \item{tracking}{}
+#'   \item{url}{}
+#'   \item{poll_id}{}
+#'   \item{question_id}{}
+#'   \item{created_date}{}
+#'   \item{timestamp}{}
+#' }
+#' @source See \url{https://github.com/fivethirtyeight/data/blob/master/congress-generic-ballot/README.md}
+"generic_polllist"
+
+
+
+#' Congress Generic Ballot Polls 
+#'
+#' The raw data behind the story
+#' "Are Democrats Winning The Race For Congress?"
+#' \url{https://projects.fivethirtyeight.com/congress-generic-ballot-polls/}.
+#'
+#' @format A data frame with 751 rows representing polls and 9 variables:
+#' \describe{
+#'   \item{model_date}{}
+#'   \item{dem_estimate}{}
+#'   \item{dem_hi}{}
+#'   \item{dem_lo}{}
+#'   \item{rep_estimate}{}
+#'   \item{rep_hi}{}
+#'   \item{rep_lo}{}
+#'   \item{timestamp}{}
+#'  
+#' }
+#' @source See \url{https://github.com/fivethirtyeight/data/blob/master/congress-generic-ballot/README.md}
+"generic_topline"
```

Comment:
Since `generic_topline` and `generic_pollist` are related datasets, can you add a `@seealso` roxygen field like in `?classic_rock_raw_data` and `?classic_rock_song_list`